extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetSchemaReader.java

```diff
@@ -337,7 +337,7 @@ public Optional<Class<?>> visit(
 
             @Override
             public Optional<Class<?>> visit(final LogicalTypeAnnotation.DateLogicalTypeAnnotation dateLogicalType) {
-                return Optional.of(int.class);
+                return Optional.of(java.time.LocalDate.class);
```

Comment:
This represents a behavior change. We need to be sure it's noted in release notes as a potentially breaking change for users, and that any associated documentation is updated.